assets/queries/cloudFormation/refreshToken_is_exposed/metadata.json

```diff
@@ -3,6 +3,6 @@
   "queryName": "RefreshToken Is Exposed",
   "severity": "MEDIUM",
   "category": "Identity and Access Management",
-  "descriptionText": "RefreshToken is a plaintext string, also does not started with '{{resolve:secretsmanager:' or '{{resolve:ssm-secure:' or does not ended with '}}'",
+  "descriptionText": "Alexa ASK Skill AuthenticationConfiguration RefreshToken must be a plaintext string, start with '{{resolve:secretsmanager:' or '{{resolve:ssm-secure:' and end with '}}'.",
```

Comment:
> must be a plaintext string,

it shouldn't be a plaintext string, we should check if's being properly managed by using a secret, so we should remove this part

Reply:
fixed